### ISSUE TITLE
[BUG] Markdown documentation does not allow tables

### ISSUE THREAD

**Issue by saschabrockel** [pawelmalak/snippet-box#42]
When I want to write down a table in the snipping documentation I'm not able to use the markdown syntax to write a table. I don't know why. To give an example:

|Spring Boot Configuration | Class or Java property files  |
|--------------------------|---|
|The Main Class | [DatabaseHubApplication](https://github.com/xy/xy/blob/master/xy/src/main/java/com/xyz/xy/DatabaseHubApplication.java) |
|Properties Files | [application.properties](https://github.com/xy/xy/blob/master/xy/src/main/resources/application.properties) |

**Comment by JohanPotgieter** [comment]
Yeah, I would also like to see the extended Markdown Syntax added to your wonderful utility.
